CONTRIBUTING.md

```diff
@@ -94,11 +98,45 @@ If your code example submission is missing any of these things, we might ask you
 
 Although many older code examples in this repo don't contain all of these things, we're working to ensure that all newer ones do.
 
-## Finding contributions to work on
+### Default branch name change
```

Comment:
This is necessary because the default branch for other GitHub sites is usually still master, right?

Reply:
I'd say it's a toss up, but either way the repo makes it clear when branch is default. 

To me, this was to prevent confusion for people who were actively developing on a fork at the time of the default branch name change, which would have broken their setup. Now, that's no longer relevant.